src/main/java/agh/ics/oop/SimulationEngine.java

```diff
@@ -0,0 +1,25 @@
+package agh.ics.oop;
+
+public class SimulationEngine implements IEngine {
+
+    private final MoveDirection[] DIRECTIONS;
```

Comment:
tu również - to zwykły atrybut, nie stała (stałe łatwo poznać po tym, że są `static final`)